docs/tutorials/multiresolution.ipynb

```diff
@@ -257,14 +251,11 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# For better detections, turn obs into obs_detect in sources\n",
-    "sources = [\n",
-    "    scarlet.ExtendedSource(frame_detect, (ra[i], dec[i]), obs, \n",
-    "                           symmetric=False, \n",
-    "                           monotonic=True, \n",
-    "                           obs_idx=1)\n",
-    "    for i in range(ra.size)\n",
-    "]\n",
+    "# For better detections, turn obs in sources. New initialisation scheme.\n",
+    "sources = initialise(frame, \n",
+    "                     obs, \n",
+    "                     [(r, d) for r, d in zip(ra, dec)], \n",
+    "                     scarlet.Extended(thresh = 5, min_grad = 0.1), coadd_sed = False)\n",
```

Comment:
I see this as a way to share the coadd, alright. But the API is clunky (see below the whole slew of empty code in `sources.py` that's needed to support this. It's also not transparent and impractical to the user. How would this look when there's a mix of source types? You'd have to manually sync the coordinate and source lists (plus all of their respective argument). In essence, you're trying to hide the very thing that needs to be exposed to the user, namely what they are fitting.

Reply:
It is not that much more code in sources.py. We could get rid of the "Setter" classes if we decided to change the API and stopped supporting the current intialisations actually. Also, I'm open to suggestions that could replace these classes. I couldn't find a better way.

Look at the initialisation in quickstart to see what it looks like when there are multiple sources. Honestly between this and and the `for ... if ... elif ... else` of the current implementationI find this one easier to. read and understand. Indeed, I am not fond of the fact that coordinates and sources have to match, but you also have to do this in the current implementation and on top of that you have to place the right coordinates as argument of EACH source. I don't see how any of your comments do not apply to the current initialisation scheme.

Reply:
The idea behind this is that I see in the future the possibility of passing a catalog that contains source type. At some point, we [or someone] will have to make an informed decision about what type of source each source. With this implementation, we could imagine a detection method that spits coordinates and source types to be fed directly to the initialise function.

Reply:
Removed